README.md

```diff
@@ -0,0 +1,121 @@
+# web3j Sample Project
+
+This project provides a very simple starter application for working with web3j and the test 
+Ethereum network (testnet) known as Rinkeby.
+
+The demo deploys a *hello world* smart contract which contains a greeting message. The current 
+greeting message can be read and modified in the smart contract.  
+
+In order to run this demo, three pieces of information are required:
+
+1. Ethereum client (or node) endpoint. The simplest thing to do is 
+[request a free access token](https://infura.io/register.html) from Infura.
+2. A wallet file. This can be generated using the web3j 
+[command line tools](https://docs.web3j.io/command_line.html) by running the command:
+3.  Some Ether. This can be requested from the [Rinkeby Faucet](https://www.rinkeby.io/#faucet).
+
+- To create a wallet file, run the command:
+```aidl
+web3j wallet create
+```
+
+- This will ask for a password, the destination directory to be placed and will be in a `.json` format.
+On the `Application.java` class under the package structure `org.web3j.sample`,
+on line `66-67`, replace the placeholders with your wallet file password and directory path.
+
+- With the access token retrieved from Infura, on line `58` replace the placeholder 
+in the url with your token.
+
+Once these details have been added to the main Application class, you'll be good to go.
+
+### Run the project
+- Make sure to be in the root directory
+- First we need to create the smart contracts deriving from the solidity file `Greeter.sol` by running:
+```aidl
+mvn web3j:generate-sources
+```
+- This will create the Greeter and Mortal contract under the package structure `org.web3j.model`
+- Clean and build the project, run the command:
+```aidl
+mvn clean install
```

Comment:
If we run `clean` we are deleting the previously generated code.

Reply:
Changed to `mvn install`